lib/Configuration.php

```diff
@@ -52,6 +52,7 @@ class Configuration
             'languageselection'        => false,
             'languagedefault'          => '',
             'urlshortener'             => '',
+            'urlshortenerajax'         => '',
```

Comment:
default may be "false" or so and not an empty string?

Reply:
data attributes are interpreted as strings so empty is false, everything else is true